src/Service/CalendarService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Baikal\Service;
+
+use Baikal\Domain\User;
+use Sabre\CalDAV\Backend\BackendInterface as CalBackend;
+
+/**
+ * UserRepository implementation using PDO
+ */
+class CalendarService {
+
+    /**
+     * @var CalBackend
+     */
+    private $calBackend;
+
+    function __construct(CalBackend $calBackend) {
+
+        $this->calBackend = $calBackend;
+        
+    }
+
+    /**
+     * Creates a new Calendar for a new User
+     */
+    function createDefault(User $user) {
+
+        $this->calBackend->createCalendar($user->getPrincipalUri(), 'default', [
```

Comment:
I would like to provide 3 default calendars:

1. Home - `VEVENT` only.
2. Work - `VEVENT` only`.
3. Tasks - `VTODO` only.